extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergUtils.java

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+public class IcebergUtils {
```

Comment:
Could be package private.

Reply:
Okay, move the comparator to somewhere else then. Could be top level package private.

Reply:
I made the comparator instance public and comparator itself private, since we need to access the instance from IcebergTableParquetLocationKey. Let me know if this is better.